jellyfin-api/src/main/kotlin/org/jellyfin/sdk/api/client/error/ApiClientError.kt

```diff
@@ -0,0 +1,8 @@
+package org.jellyfin.sdk.api.client.error
+
+/**
+ * Base exception for the ApiClient.
+ */
+public abstract class ApiClientError(
+	message: String?
+) : Error(message)
```

Comment:
Could be single-line. If not, should have a trailing comma. Also why `Error` and not `Exception`?
Same for the other error classes ofc.

Reply:
Should be exception, I'll change it.

Reply:
also, multi-line for readability